Projects/UOContent/Mobiles/Townfolk/Banker.cs

```diff
@@ -103,6 +109,51 @@ public static int GetBalance(Mobile m, out List<Item> gold, out List<Item> check
         return (int)Math.Clamp(balance, 0, int.MaxValue);
     }
 
+    private static bool HasRequiredBalance(int requiredBalance, Mobile m, out PooledRefList<Gold> gold, out PooledRefList<BankCheck> checks)
+    {
+        Container bank = m.FindBankNoCreate();
+
+        if (bank == null)
+        {
+            gold = default;
+            checks = default;
+
+            return false;
+        }
+
+        long balance = 0;
+
+        gold = PooledRefList<Gold>.Create();
+        checks = PooledRefList<BankCheck>.Create();
```

Comment:
Let's move this to LIne 140 (above the bankcheck foreach).
Then above line 136 (the return true), let's add `checks = default;`